languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -46469,6 +46469,73 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
     <!-- Grammatik -->
     <!-- ====================================================================== -->
     <category id="GRAMMAR" name="Grammatik">
+        <rule id="FALSCHER_IMPERATIV" name="Machst du (Mach|Mache) die Tür zu." default="temp_off">
+            <antipattern>
+                   <token postag="SENT_START"/>
+                <marker>
+                    <token postag="VER:2:SIN.*" postag_regexp="yes"/>
+                    <token>du</token>
+                </marker>
+                <token skip="-1"/>
+                <token>?</token>
+            </antipattern>
+            <antipattern>
+                   <token postag="SENT_START"/>
+                <marker>
+                    <token postag="VER:2:SIN.*KJ2.*" postag_regexp="yes"/>
+                    <token>du</token>
+                </marker>
+                <token skip="-1"/>
+                <token>.</token>
+            </antipattern>
+            <antipattern>
+                   <token postag="SENT_START"/>
+                <marker>
+                    <token postag="VER:2:SIN.*" postag_regexp="yes"/>
+                    <token>du</token>
+                </marker>
+                <token skip="-1"/>
+                <token>,</token>
+            </antipattern>
+            <antipattern>
+                   <token postag="SENT_START"/>
+                <marker>
+                    <token postag="VER:2:SIN.*" postag_regexp="yes"/>
+                    <token>du</token>
+                    <token min="0" postag="EIG.*" postag_regexp="yes"/>
+                </marker>
+                <token regexp="yes">,|:|;|–</token>
+            </antipattern>
+            <antipattern>
+                <token postag="SENT_START"/>
+                <marker>
+                    <token postag="VER:2:SIN.*" postag_regexp="yes"/>
+                    <token>du</token>
+                </marker>
+                <token skip="-1"/>
+                <token postag="ABK.*" postag_regexp="yes"/>
+                <token>.</token>
+            </antipattern>
+            <pattern>
+                <token postag="SENT_START"/>
+                <marker>
+                    <token postag="VER:2:SIN.*" postag_regexp="yes"/>
+                    <token>du</token>
+                </marker>
+                <token skip="-1"/>
```

Comment:
Hier auch wieder die Sache mit `skip="-1"`. Wenn du es in das token "du" darüber machst, brauchst du hier kein dummy-token.

Außerdem hat jeder Satz ein SENT_END. Wenn dazwischen eh alles Mögliche stehen kann, brauchst du das im Pattern nicht zu erwähnen

Reply:
Ich habe soweit alles angepasst. wenn es keine weiteren Vorschläge/Kritik gibt, kann gemerged werden.